sale_order_invoicing_finished_task/i18n/es.po

```diff
@@ -0,0 +1,86 @@
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 1.8.7.1\n"
+
+#. module: sale_order_invoicing_finished_task
+#: model:ir.model.fields,help:sale_order_invoicing_finished_task.field_product_product__invoicing_finished_task
+#: model:ir.model.fields,help:sale_order_invoicing_finished_task.field_product_template__invoicing_finished_task
+#: model:ir.model.fields,help:sale_order_invoicing_finished_task.field_project_task__invoicing_finished_task
+msgid "Invoice the order lines only when the task is set to invoiceable"
+msgstr ""
+"Se facturan las lineas de pedido solo cuando la tarea esta marcada como "
+"facturable"
+
+#. module: sale_order_invoicing_finished_task
+#: model:ir.model.fields,field_description:sale_order_invoicing_finished_task.field_project_task__invoiceable
+#: model:ir.model.fields,field_description:sale_order_invoicing_finished_task.field_project_task_type__invoiceable
+msgid "Invoiceable"
+msgstr "Facturable"
+
+#. module: sale_order_invoicing_finished_task
+#: model:ir.model.fields,field_description:sale_order_invoicing_finished_task.field_product_product__invoicing_finished_task
+#: model:ir.model.fields,field_description:sale_order_invoicing_finished_task.field_product_template__invoicing_finished_task
+#: model:ir.model.fields,field_description:sale_order_invoicing_finished_task.field_project_task__invoicing_finished_task
+msgid "Invoicing control by task"
+msgstr "Control de facturación por tarea"
+
+#. module: sale_order_invoicing_finished_task
+#: model:ir.model,name:sale_order_invoicing_finished_task.model_product_template
+msgid "Product Template"
+msgstr "Plantilla de producto"
+
+#. module: sale_order_invoicing_finished_task
+#: model:ir.model,name:sale_order_invoicing_finished_task.model_sale_order
+#, fuzzy
+msgid "Sale Order"
+msgstr "Línea de pedido de venta"
+
+#. module: sale_order_invoicing_finished_task
+#: model:ir.model,name:sale_order_invoicing_finished_task.model_sale_order_line
+msgid "Sales Order Line"
+msgstr "Línea de pedido de venta"
+
+#. module: sale_order_invoicing_finished_task
+#: model:ir.model,name:sale_order_invoicing_finished_task.model_project_task
+msgid "Task"
+msgstr "Tarea"
+
+#. module: sale_order_invoicing_finished_task
+#: model:ir.model,name:sale_order_invoicing_finished_task.model_project_task_type
+msgid "Task Stage"
+msgstr "Etapa de la tarea"
+
+#. module: sale_order_invoicing_finished_task
+#: model:ir.model.fields,field_description:sale_order_invoicing_finished_task.field_sale_order_line__task_ids
+msgid "Tasks"
+msgstr "Tareas"
+
+#. module: sale_order_invoicing_finished_task
+#: code:addons/sale_order_invoicing_finished_task/models/project.py:65
+#, python-format
+msgid ""
+"You cannot create/modify a task related with a invoiced, done or cancel sale "
+"order line "
+msgstr ""
+"No puede crear/modificar una tarea relacionada con una línea de pedido "
+"facturada, realizada o cancelada"
+
+#~ msgid "Quotation"
+#~ msgstr "Presupuesto"
```

Comment:
```suggestion
```

Reply:
I think it is better to let @oca-transbot update this files and translate them later for consistency.